protocol/communities/community.go

```diff
@@ -162,11 +171,11 @@ func (o *Community) MarshalPublicAPIJSON() ([]byte, error) {
 		}
 
 		communityItem.TokenPermissions = o.tokenPermissions()
+		communityItem.PendingAndBannedMembers = o.PendingAndBannedMembers()
```

Comment:
I don't think we want to store this here. We're polluting the `CommunityDescription` with information that is only relevant for admins and the control node.

I think what should rather happen is:

1. Create an new table for community members that tracks their status
2. Update a member's entry state when admin marks as kicked/ban pending
3. Send that info to control node and other admins
4. Other admins will update their own table accordingly (also want to store **who** made that decision, so someone who's done a decision can undo it again)
5. Control node will process the pending state message and sends decision to other admins

^ Notice that the pending state here never needed to be processed or seen by non-admin member nodes.

Reply:
We do not want to make the banned members public (so not part of the description)

Reply:
To be fair, we already have currently banned members in the `BanList` which lives in the community description.
We should likely remove it from there and use the same mechanism as described above for sharing this information with other admins.

Reply:
1. All right, I'll get it out of here. I put it here because the bannedList was here originally, but if we don't show banned users to regular community members, I'll remove it.
2.  I relied on https://github.com/status-im/status-go/pull/3925 to implement this PR. Pending states completely rely on the state obtained from community events and I think a separate table is not needed here. If the ability to cancel requests is needed, it seems reasonable to me to add a new community event that will remove another specified community event from the queue. I would prefer to do this in a different PR.